Jenkinsfile-smoke.groovy

```
@@ -0,0 +1,27 @@
+/*
+* Requires: https://github.com/RedHatInsights/insights-pipeline-lib
+*/
+
+@Library("github.com/RedHatInsights/insights-pipeline-lib@v3") _
+
+if (env.CHANGE_ID) {
+    execSmokeTest(
+        // the service-set/component for this app in e2e-deploy "buildfactory"
+        ocDeployerBuilderPath: "xavier/xavier-integration",
+    
+        // the service-set/component for this app in e2e-deploy "templates"
+        ocDeployerComponentPath: "xavier/xavier-integration",
+    
+        // the service sets to deploy into the test environment
+        ocDeployerServiceSets: "xavier/xavier-integration",
```

This is the set of services the `xavier-integration` application need to run.
```suggestion
        ocDeployerServiceSets: "platform-mq,rbac,ingress,xavier",
```